packages/components/src/components/ui-shell/_side-nav.scss

```diff
@@ -22,7 +22,7 @@
 ///   }
 @mixin experimental-focus() {
   outline: 4px solid $shell-side-nav-accent-01;
-  outline-offset: -4px;
+  outline-offset: rem(-4px);
```

Comment:
I removed this entire block in my other PR that is up, and replaced the `experimental-focus` with our main focus mixin since we are using the standard 2px here as well.